packages/frameworks/frameworks.json

```diff
@@ -700,13 +700,13 @@
     },
     "settings": {
       "buildCommand": {
-        "value": "yarn rw db up --no-db-client --auto-approve && yarn rw build"
+        "value": "yarn rw build && yarn rw db up --no-db-client --auto-approve && yarn rw dataMigrate up"
```

Comment:
This is the command recommended here: https://github.com/redwoodjs/create-redwood-app/blob/21960fe7cd9c41c1546d239c6950a6db4199b6f5/netlify.toml#L2